api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -582,104 +579,6 @@ public void testMe_userBeforeNotLoggedInSuccess() {
         false);
   }
 
-  @Deprecated // to be removed in RW-4362
```

Comment:
none of these are applicable to the Verified version

Reply:
Do we have equivalent or better coverage of the new stuff?

Reply:
Do we have equivalent coverage of the new stuff?

Reply:
These are about lists of affiliations - no equivalent concept exists.

Equivalent feature coverage for the general concept of affiliation already exists